docs/displaysystem/angular.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: "Angular Display System"
+description: ""
+group: display
+---
+{% include JB/setup %}
+
+
+### Angular (Beta)
+
+Angular display system treats output as an view template of [AngularJS](https://angularjs.org/).
+It compiles templates and display inside of Zeppelin.
+
+Zeppelin provides gateway between your interpreter and your compiled AngularJS view teamplates.
+Therefore, you can not only update scope variable from your interpreter  but also watch your scope variable in the interpreter, which is JVM process.
+
+<br />
+#### Print AngularJS view
+
+To use angular display system, your output should starts with "%angular".
+<img src="../../assets/themes/zeppelin/img/screenshots/display_angular.png" width=600px />
+
+Note that display system is backend independnet.
+
+Because of variable 'name' is not defined, 'Hello \{\{name\}\}' display 'Hello '.
+
+<br />
+#### Bind/Unbind variable
+
+Through ZeppelinContext, you can bind/unbind variable to AngularJS view.
+
+
+```
+// bind my 'object' as angular scope variable 'name' in current notebook.
```

Comment:
About syntax, i was trying to show usage example rather than showing
definition of method.

Didn't know about java repl https://github.com/albertlatacz/java-repl,
interesting. Thanks for letting me know.

Best,
moon

On Sun, Aug 9, 2015 at 9:22 PM Felix Cheung <notifications@github.com>
wrote:

> In docs/displaysystem/angular.md
> <https://github.com/apache/incubator-zeppelin/pull/188#discussion_r36601183>
> :
>
> > +
> > +To use angular display system, your output should starts with "%angular".
> > +<img src="../../assets/themes/zeppelin/img/screenshots/display_angular.png" width=600px />
> > +
> > +Note that display system is backend independnet.
> > +
> > +Because of variable 'name' is not defined, 'Hello \{\{name\}\}' display 'Hello '.
> > +
> > +<br />
> > +#### Bind/Unbind variable
> > +
> > +Through ZeppelinContext, you can bind/unbind variable to AngularJS view.
> > +
> > +
> > +```
> > +// bind my 'object' as angular scope variable 'name' in current notebook.
>
> Yap. I didn't realize there isn't Java REPL (maybe we should?
> https://github.com/albertlatacz/java-repl)
> In such case should they be in the method definition in Scala syntax
> instead? eg.
>
> def angularBind(name: String, object: Object): Unit
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/188/files#r36601183>.
>